docs/features.md

```diff
@@ -0,0 +1,55 @@
+# pymwp Features
+
+## Supported C-Language Features
+
+This section provides a summary of supported C language features pymwp can 
+analyze. It also lists language features that are in the process of being 
+implemented and for which implementation is planned.
+
+!!! info "Note on C file parsing"
+    pymwp uses pycparser to process the C input file. Any limitation of 
+    pycparser is also a limitation in pymwp. From
+    [pycparser website](https://github.com/eliben/pycparser):
+    "pycparser aims to support the full C99 language (according to the 
+    standard ISO/IEC 9899). Some features from C11 are also supported."
+  
+**Legend**
+
+- ✅ &nbsp; **ready** - fully implemented and ready to use
+- 🟧 &nbsp; **in progress** - implementation is in progress but not ready
+- 🟥 &nbsp; **planned** - implementation is in a planning stage
+- ⬜ &nbsp; **future**
+
+Description | State | Example
+--- | --- | ---
+**Basic data types** ||
+ Integer types (incl. `signed`, `unsigned`) | ✅ | `char`, `short`, `int`, `long`, 
+ `long long`
```

Comment:
I wouldn't say that `char` is an integer type, is it?
It is interesting that the analysis supports more datatype (typically, char), but I'm not sure how to interpret the result of the analysis in those cases.

Reply:
[C reference](https://en.wikipedia.org/wiki/C_data_types#Main_types) they say it is integer type.